src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2388,6 +2388,236 @@ curl -X POST \
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.5/configuration/action/create_tag \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "name": "docs_feedback",
+    "group_ids": [
+      0,
+      42
+    ]
+  }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Delete Tag
+
+Deletes an existing tag.
+
+#### Specifics
+
+|                 |                                                                    |
+| --------------- | ------------------------------------------------------------------ |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/delete_tag` |
+| Required scopes | `tags--all:rw, tags--groups:rw`                                    |
+
+#### Request
+
+| Parameter | Required | Data type | Notes                                                   |
+| --------- | -------- | --------- | ------------------------------------------------------- |
+| `name`    | yes      | `string`  | Matching the name of an existing tag is case-sensitive. |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.5/configuration/action/delete_tag \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "name": "docs_feedback"
+  }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### List Tags
+
+Lists the exisiting tags.
+
+#### Specifics
+
+|                 |                                                                   |
+| --------------- | ----------------------------------------------------------------- |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/list_tags` |
+| Required scopes | `tags--all:ro, tags--groups:ro`                                   |
+
+#### Request
+
+| Parameter           | Required | Data type | Notes                                                                  |
+| ------------------- | -------- | --------- | ---------------------------------------------------------------------- |
+| `filters`           | No       | `object`  | Possible request filters.                                              |
+| `filters.group_ids` | Yes      | `int[]`   | IDs of the groups from which you want to list Tags.                    |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.5/configuration/action/list_tags \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "filters": {
+      "group_ids": [
+        0,
+        42
+      ]
+    }
+  }'
+```
+
+</CodeSample>
+
+<CodeResponse type="configuration" version="v3.5" json="listTags"/>
```

Comment:
This is not rendered. Export it in `payloads/management/v3.5/configuration-api/index.js`